ansible_collections/arista/cvp/molecule/fixtures/cv_device_v3.yaml

```diff
@@ -1,4 +0,0 @@
-- node: 192.168.0.5
```

Comment:
is this desired?

Reply:
I think I answered my own question, and it looks like we can safely remove this since this is not in use anymore.